src/fides/api/ops/api/v1/endpoints/config_endpoints.py

```diff
@@ -33,7 +33,7 @@ def get_config(
     logger.info("Getting the exposable Fides configuration")
     if api_set:
         logger.info("Retrieving api-set application settings")
-        return ApplicationConfig.get_api_set_config(db)
+        return ApplicationConfig.get_api_set(db)
     config = censor_config(get_app_config())
```

Comment:
This is pulling the config in the more traditional way instead of from the database which is fine, this is just making me think more about the dangers of having two sources of truth when something gets out of sync.

Reply:
Changed my mind, we're largely only changing the config in tests though, it's not a big deal.